pom.xml

```diff
@@ -148,6 +148,7 @@
                 <global_storm_kafka_version>1.2.3</global_storm_kafka_version>
                 <global_zeppelin_version>0.8.0</global_zeppelin_version>
                 <global_solr_version>7.4.0</global_solr_version>
+                <base_hadoop_version>3.1.1</base_hadoop_version>
```

Comment:
In this pom, outside of the HDP-3.1 profile, we still have...
```
<base_hadoop_version>2.7.1</base_hadoop_version>
<global_hadoop_version>${base_hadoop_version}</global_hadoop_version>
```
And then inside the HDP-3.1 profiler we redefine...
```
<base_hadoop_version>3.1.1</base_hadoop_version>
```
This all seems confusing to me.

Can we just follow what we did for the other versions like Zookeeper, Curator, HBase, etc?  We could just define a single `global_hadoop_version` in each profile and then get rid of this `base_hadoop_version` business altogether.
```
<global_hadoop_version>3.1.1</global_hadoop_version>
```

And ultimately before we merge this feature branch into master, we will remove the HDP-2.5.0.0 profile in the POM.

Reply:
Actually, I think we can leave this as-is here.  I was looking through this business of base and global versions and we have a few other things we need to clean-up here also prior to merging this into master.  It probably makes sense to leave this as a follow-on PR.

Reply:
> Actually, I think we can leave this as-is here. I was looking through this business of base and global versions and we have a few other things we need to clean-up here also prior to merging this into master. It probably makes sense to leave this as a follow-on PR.

I had debated how we should handle this as well. I'd probably be ok with another final FB cleanup PR here.